src/spu-schema/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-spu-schema"
-version = "0.7.0"
+version = "0.8.0"
```

Comment:
does it require major bump?

Reply:
I think so, because `wasm_payload` was added to `StreamFetchRequest` in https://github.com/infinyon/fluvio/pull/1327

Reply:
k